onadata/settings/common.py

```diff
@@ -509,6 +510,31 @@ def configure_logging(logger, **kwargs):
     'application/zip',
 ]
 
+CELERY_BEAT_SCHEDULE = {
+    'Mark-Expired-Exports-Failed': {
+        'task':
+        'onadata.apps.viewer.tasks.mark_expired_pending_exports_as_failed',
+        'schedule': crontab(hour='*', minute='0'),
+        'options': {'ignore_result': True}
+    },
+    'Delete-Expired-Failed-Exports': {
+        'task': 'onadata.apps.viewer.tasks.delete_expired_failed_exports',
+        'schedule': crontab(hour='*', minute='10'),
+        'options': {'ignore_result': True}
+    },
+    'celery.backend_cleanup': {
+        'task': 'celery.backend_cleanup',
+        'schedule': crontab(hour='*', minute='30'),
+        'options': {'ignore_result': True}
+    },
+    'Delete-Inactive-Users': {
+        'task': 'onadata.apps.api.tasks.delete_user_async',
+        'schedule': crontab(
+            0, 0, day_of_month='30'),  # Execute on the 30th of every month.
+        'options': {'ignore_result': True}
+    },
+}
+
```

Comment:
I don't think this configuration should be placed within the `common` settings..... Celery beat scheduling seems like something the user should configure within their own local setting IMO

Reply:
Places this here since the tasks are things that should run in all environments/deployments of onadata

Im unsure which settings file would be a better fit for this configurations.
Cc: @ivermac